docs(popover): update doc (#174)
lumapps/design-system: packages/site-demo/content/product/components/popover/index.mdx

# Popover

> **Popovers display pieces of UI on top of others. They are positioned from anchor elements.**

Reviewer: ```suggestion
**Popovers display pieces of UI on top of others. They are positioned using anchor elements.**
```

## Default

<DemoBlock demo="default" />

## Positions

### Directed

Popovers can be placed to 4 positions :
_TOP_, _RIGHT_, _BOTTOM_, _LEFT_

<DemoBlock demo="positions" />

### Auto

> Popovers can also be placed automatically where space is available. They will be placed at the bottom of anchors by default.

Reviewer: ```suggestion
Popovers can also be placed automatically where space is available. They are be placed at the bottom of anchors by default.
```
It's always better to avoid using the future tense.

> Popovers can also be placed automatically where space is available. They are be placed at the bottom of anchors by default.

Reviewer: ```suggestion
Popovers can also be placed automatically where space is available. They are placed at the bottom of anchors by default.
```

sorry there was a typo in my review...

<DemoBlock demo="auto" />

## Offset

Popovers positions can have an offset.

<DemoBlock demo="offset" />

## Alignments

In addition to positions, popovers can be aligned to a chosen anchor's side :
_TOP_START_, _TOP_END_, _RIGHT_START_, _RIGHT_END_, _BOTTOM_START_, _BOTTOM_END_, _LEFT_START_, _LEFT_END_, _AUTO_START_, _AUTO_END_,

<DemoBlock demo="alignments" />

## Matching width

> Popovers' width can be forced to match anchor's.

Reviewer: ```suggestion
Popovers width can be forced to match anchors.
```
Let's simplify our life :)
